rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -761,3 +807,84 @@ def test_validator_daily_stats_with_db_interaction(  # pylint: disable=unused-ar
         )
         assert stats_call.call_count == 1
         assert stats[:len(expected_stats)] == expected_stats
+
+
+@pytest.mark.parametrize('default_mock_price_value', [FVal(1.55)])
+def test_validator_with_custom_ownership(  # pylint: disable=unused-argument  # noqa: E501
```

Comment:
no no no. Do not add another new test that creates a whole new api server, scrapes for multiple validators (this is super slow) just to test ownership. 

Add a small part in the test above to test it.

What you should test is marking the ownership as 0.45% (or whatever) of a validator after it has had its daily stats removed.